releasenotes/notes/add-light-cone-pass-6c56085734512e98.yaml

```
@@ -0,0 +1,53 @@
+---
+features_transpiler:
+  - |
+    Added a new transpiler pass, :class:`.LightCone` that is used
+    to get the lightcone of a circuit when measuring a subset of
+    qubits or a specific Pauli string.
+    For example if you had a circuit like:
+
+        .. plot::
+
+          from qiskit.transpiler.passes.optimization.light_cone import LightCone
+          from qiskit.transpiler.passmanager import PassManager
+          from qiskit.circuit import QuantumCircuit
+
+          qc = QuantumCircuit(3,1)
+          qc.h(0)
+          qc.h(1)
+          qc.h(2)
```

this could be written more concise if you want
```suggestion
          qc.h(range(3))
```